cloudformation.properties

```diff
@@ -1,14 +1,33 @@
-name=Cloudformation Plugin for SonarQube
+name=Cloudformation
 category=External Analysers
-descripton=Cloudformation rules for json and yaml, supporting cfn-nag
+descripton=Cloudformation rules for json and yaml, integrates cfn-nag reports.
 homepageUrl=https://github.com/Hack23/sonar-cloudformation-plugin
-publicVersions=2.0.1
+archivedVersions=2.0.1,2.0.2,2.0.3
+publicVersions=2.0.4
 
 defaults.mavenGroupId=com.hack23.sonar
 defaults.mavenArtifactId=sonar-cloudformation-plugin
 
+2.0.4.description=Change name to Cloudformation.
+2.0.4.date=2020-04-09
+2.0.4.sqVersions=[7.9,LATEST]
+2.0.4.changelogUrl=https://github.com/Hack23/sonar-cloudformation-plugin/releases/tag/sonar-cloudformation-plugin-2.0.4
+2.0.4.downloadUrl=https://oss.sonatype.org/service/local/repositories/releases/content/com/hack23/sonar/sonar-cloudformation-plugin/2.0.4/sonar-cloudformation-plugin-2.0.4.jar
+
+2.0.3.description=Support new cfn-nag rule, W70 Cloudfront should use minimum protocol version TLS 1.2.
+2.0.3.date=2020-04-09
+2.0.3.sqVersions=[7.9,8.2]
+2.0.3.changelogUrl=https://github.com/Hack23/sonar-cloudformation-plugin/releases/tag/sonar-cloudformation-plugin-2.0.3
+2.0.3.downloadUrl=https://oss.sonatype.org/service/local/repositories/releases/content/com/hack23/sonar/sonar-cloudformation-plugin/2.0.3/sonar-cloudformation-plugin-2.0.3.jar
```

Comment:
I guess you're listing these straight-to-archive versions for the changelog (kudos and smart thinking!). FYI there's no point in including a downloadUrl for them since the Marketplace will never offer them for download. (And I guess you had no way of knowing that.)